src/test/java/core/basesyntax/sevrice/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.sevrice.impl;
+
+import static org.junit.Assert.fail;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.impl.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.sevrice.FruitService;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.strategyimpl.BalanceOperationHandler;
+import core.basesyntax.strategy.strategyimpl.OperationStrategyImpl;
+import core.basesyntax.strategy.strategyimpl.PurchaseOperationHandler;
+import core.basesyntax.strategy.strategyimpl.ReturnOperationHandler;
+import core.basesyntax.strategy.strategyimpl.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
```

Comment:
You're testing logic here that doesn't belong to the FruitService class. Test every option of strategy separately